src/frontend-pwa/src/views/Report/report.styles.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @summary This is the styling page for the Report view screen.
+ * @author  Tyler Maloney
+ * */
+
```

Comment:
At this stage of development we will need to nicely style this before it hits production instead of adding future tickets to correct it.
Also using iPhone 6/7/8 the width and height are oversized to the screen